CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ option(BUILD_VINEYARD_FUSE "Enable vineyard's fuse support" OFF)
 option(BUILD_VINEYARD_FUSE_PARQUET "Enable vineyard's fuse parquet support" OFF)
 option(BUILD_VINEYARD_HOSSEINMOEIN_DATAFRAME "Enable hosseinmoein dataframe support" OFF)
 option(BUILD_VINEYARD_LLM_CACHE "Enable kv-state cache support" ON)
+option(BUILD_VINEYARD_RDMA "Enable RDMA support" OFF)
```

Comment:
Build RDMA support by default.

Reply:
Libfabric may not be on the build environment. Make it as a submodule?